lms/static/coffee/src/instructor_dashboard/student_admin.coffee

```diff
@@ -171,6 +180,83 @@ class StudentAdmin
           create_task_list_table @$table_task_history_single, data.tasks
         error: std_ajax_err => @$request_response_error_grade.text full_error_message
 
+   # reset entrance exam attempts for student
+    @$btn_reset_entrance_exam_attempts.click =>
+      unique_student_identifier = @$field_entrance_exam_student_select_grade.val()
```

Comment:
Please add Jasmine tests for all of the new logic here. Unfortunately CoffeeScript code isn't counted by code coverage so this doesn't show up as a gap in the reports.

Ideally, all new client-side logic would be written as Backbone views in JavaScript. We are starting to move the instructor dashboard this way with the membership tab. For this small a change, it is okay to go with what you have. For the future, see: https://openedx.atlassian.net/wiki/display/TNL/Code+Review+Guidelines+and+Gotchas